server/db/index.js

```diff
@@ -1,10 +1,19 @@
 const mongoose = require('mongoose');
 
-mongoose
-  .connect('mongodb+srv://user:password@lobstericecream-trflp.gcp.mongodb.net/home?retryWrites=true&w=majority')
-  .catch((e) => {
-    console.error('Connection error', e.message);
-  });
+const defaultUri = 'mongodb+srv://user:password@lobstericecream-trflp.gcp.mongodb.net/home?retryWrites=true&w=majority';
```

Comment:
Ideally you don't want to store the password in Github. One option is to store it in a dotenv file.

Reply:
Added the dotenv solution